include/cantera/oneD/StFlow.h

```diff
@@ -432,6 +462,17 @@ class StFlow : public Domain1D
     //! to `j1`, based on solution `x`.
     virtual void updateTransport(doublereal* x, size_t j0, size_t j1);
 
+    // support continuation procedure	
+    static bool m_strainRateEq;
+    bool m_UnityLewisNumber;
+    static bool m_onePointControl;
+    static bool m_twoPointControl; 
+    doublereal m_Tfuel;
+    int m_Tfuel_j;
+    doublereal m_Toxid; 
+    int m_Toxid_j;
+    static bool m_reactions;
```

Comment:
@speth I've cloned this repo and am trying to make the changes that you requested. For this comment, do you specifically mean just these 4 lines, or all of the variables in the block that these lines are a part of? Some other variables above these are also defined as static, and I just wanted to be sure that removing the 'static' identifier on those would be ok.

Reply:
@wandadars This PR has been superseded by #534, suggest making changes over there

Reply:
@bryanwweber Oh. I must have got lost. I'm on the #534 PR. I guess the "outdated" tag on a comment means that it is from an older/different PR?

Reply:
@wandadars Oh, sorry, I meant superseded by #541 The `Outdated` means that the code that is specifically referenced in this comment has been modified in a later commit on this branch.

Reply:
The comment applies to more than just the 4 lines shown (which is a choice made by Github, not me). Many of these comments are on just a specific instance of an issue that occurs in multiple locations -- I'm not going to go through and comment on ever single indentation issue, for example.

And yes, all changes should be built on top of #541 rather than this PR. Thanks for picking this up.